crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -657,6 +682,71 @@ pub fn process_remote_insert_request(
     Ok(Value::Null)
 }
 
+/// Handles a `bevy/mutate_component` request coming from a client.
+///
+/// This method allows you to mutate a single field inside an Entity's
+/// component.
+pub fn process_remote_mutate_component_request(
+    In(params): In<Option<Value>>,
+    world: &mut World,
+) -> BrpResult {
```

Comment:
I feel like the method handler could probably be done better. Any pointers here would be appreciated.